src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -1211,8 +1288,7 @@ public SchemaShape GetOutputSchema(SchemaShape inputSchema)
         public ImageClassificationTransformer Fit(IDataView input)
         {
             _host.CheckValue(input, nameof(input));
-            if (_transformer == null)
-                _transformer = new ImageClassificationTransformer(_host, _options, _dnnModel, input);
+            _transformer = new ImageClassificationTransformer(_host, _options, _dnnModel, input);
```

Comment:
>_transformer [](start = 12, length = 12)

The `_transformer` field is not needed, you can just have:
`return new ImageClassificationTransformer(...)`.

Reply:
Yup, in my commit before I fixed that.